src/fmu/dataio/export/rms/volumetrics.py

```diff
@@ -54,141 +49,87 @@ class _ExportVolumetricsRMS:
     project: Any
     grid_name: str
     volume_job_name: str
-
-    # optional and defaulted
-    global_config: str | Path | dict = "../../fmuconfig/output/global_variables.yml"
-    forcefolder: str = ""  # allowed until deprecated
-    subfolder: str = ""
-    name: str = ""
-    tagname: str = "vol"
-    classification: str = "restricted"
-    workflow: str = "rms volumetric run"
-
-    # internal storage instance variables
-    _global_config: dict = field(default_factory=dict, init=False)
-    _volume_job: dict = field(default_factory=dict, init=False)
-    _volume_table_name: str = field(default="", init=False)
-    _dataframe: pd.DataFrame = field(default_factory=pd.DataFrame, init=False)
-    _units: str = field(default="metric", init=False)
+    config_path: str | Path = "../../fmuconfig/output/global_variables.yml"
+    classification: str | None = None
 
     def __post_init__(self) -> None:
         _logger.debug("Process data, estiblish state prior to export.")
-        self._check_rmsapi_version()
-        self._set_global_config()
-        self._rms_volume_job_settings()
-        self._read_volume_table_name_from_rms()
-        self._voltable_as_dataframe()
-        self._set_units()
-        self._warn_if_forcefolder()
+        self._config = _load_config(Path(self.config_path))
+        self._volume_job = self._rms_volume_job_settings()
```

Comment:
From the function name it is a bit unclear what this function (`rms_volume_job_setting`) is doing. I suggest adding a verb to the name to make it more clear

Reply:
true 🙂  I'll add it